### ISSUE TITLE
Retrieve data chunk from image client

### ISSUE THREAD

**Issue by Wanru-He** [ifm/o3d3xx-python#15]
Hi, I am working on the O3D303 camera and I hope to take pictures and analyse the data chunk on a separate python script. Is there a way to retrieve a specific image data chunk (lets say z image) from PCIC image client after a trigger is issued (to force take photos from the camera)? I am able to do this manually from the windows vision application, but I cant find a way to achieve this using this library. Thank you.

**Comment by lola-masson** [comment]
Hi @Wanru-He, sorry for the (long) delay, it appears that this issue got lost. Anything we can help you with at the moment?

**Comment by diegosalomone** [comment]
Hey, I have a similar setup where a physical trigger activates the camera and I want to have the image generated via Python. I can't find a way to do it. Can any of you help?
If I setup the TriggerMode to 1 (continuous) I can see images being generated on the python script. But with 3,4,5 (physical triggers) the python script never receives and image.

**Comment by cfreundl** [comment]
@diegosalomone Did you try the `image_grabber.py` script in the examples folder already? This one should be able to receive the images for any trigger type.

In case the script is not working: do you see the results if you connect to the camera with the Vision Assistant and execute a hardware trigger?

**Comment by diegosalomone** [comment]
Yeah, I tried. Thanks for the suggestion. Also, while exploring the inputs/outputs we can see the TriggerMode parameter (from the docs 1: free run, 2: process interface, 3: positive edge, 4: negative edge, 5: positive and negative edge).

My issue was more on the physical world, sorry. Apparently our proximity switch was too low so didn't trigger the capture. This is now fixed and working with _TriggerMode=3_.

Thanks @cfreundl . Btw, if anyone wants, the official docs: https://www.ifm.com/mounting/706397UK.pdf
